.github/workflows/test-notify-slack.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+name: 'Notify slack'
+
+on:
+  push:
+  workflow_call:
+    secrets:
+      slack_webhook_url:
+        required: true
+    # inputs:
+    #   application:
+    #     description: 'The friendly name for the application'
+    #     type: string
+    #     required: true
+    #   environment:
+    #     description: 'The environment that is being targeted for this deployment'
+    #     type: string
+    #     required: true
```

Comment:
This should be run on pull_request when specific paths are updated such as the reusable workflow or its tests.  See another test workflow for example.